.gitignore

```diff
@@ -7,4 +7,5 @@ build/
 .env
 .env*.local
 
-log/
```

Comment:
not remove .log files, becuase .log file mass storage
you add log file and commit

Reply:
ok

Reply:
ok merge with main branch

Reply:
ok!

Reply:
my adding log files

Reply:
ok my merge to main branch